open-source-trumps-innersource.md

```diff
@@ -0,0 +1,43 @@
+## Title
+Open Source trumps InnerSource
+
+## Patlet
+
+People find the InnerSource project but, after all things are considered, even if the InnerSource component meets their needs, they still go with the open source component.
+
+## Statement of Problem
+* People find the InnerSource project but, even if the InnerSource component meets their needs better than the respective open source component, they choose the open source component over the InnerSource component.
```

Comment:
This is pretty much equivalent to the **Patlet** can we deduplicate or remove parts here?

Reply:
I removed the duplication by writing a new patlet that contains both problem and solution.
Resolving.